tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/Event/__snapshots__/VenueTest__test_render_with_event_with_venue__1.php

```diff
@@ -2,6 +2,6 @@
 	<span class="tribe-events-calendar-list__event-venue-title tribe-common-b2--bold">
 		Venue 2233 title	</span>
 	<span class="tribe-events-calendar-list__event-venue-address">
-		100 Rue de Le Chat, Paris, Ile de France	</span>
```

Comment:
Can we add a second test that includes a state/province as well? To be sure it doesn't get duplicated accidentally.

Reply:
:doh: